ayon_api/server_api.py

```diff
@@ -2256,33 +2284,11 @@ def get_dependency_packages(self):
                 server.
         """
 
-        endpoint = "desktop/dependencyPackages"
-        major, minor, _, _, _ = self.server_version_tuple
-        if major == 0 and minor <= 3:
-            endpoint = "desktop/dependency_packages"
-
+        endpoint = self._get_dependency_package_route()
```

Comment:
We are not passing any arg here, so it will be always `enpoint` will always be "dependencies", no?